Facepunch.Steamworks.Test/Client/Lobby.cs

```diff
@@ -0,0 +1,330 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using Facepunch.Steamworks;
+using System.Text;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Facepunch.Steamworks.Test
+{
+    [TestClass]
+    [DeploymentItem("steam_api.dll")]
+    [DeploymentItem("steam_api64.dll")]
+    [DeploymentItem("steam_appid.txt")]
+    public class Lobby
+    {
+        [TestMethod]
+        public void CreateLobby()
+        {
+            using (var client = new Facepunch.Steamworks.Client(252490))
```

Comment:
Oh okay you can fail the test by creating a lobby with an obscene max members. Like 100000000.